sql-plugin/src/main/scala/com/nvidia/spark/rapids/SpillableColumnarBatch.scala

```diff
@@ -371,6 +371,12 @@ class SpillableHostBuffer(handle: RapidsBufferHandle,
     handle.close()
   }
 
+  def getHostBuffer(): HostMemoryBuffer = {
+    withResource(RapidsBufferCatalog.acquireBuffer(handle)) { rapidsBuffer =>
+      rapidsBuffer.getHostMemoryBuffer
+    }
+  }
+
   /**
```

Comment:
I did not remove withHostBufferReadOnly/withHostBufferWriteLock yet, but I don't think we need them if we switch to this alternate approach?

Reply:
I'd remove them if they are not useful